tests/tests_cib_resources_create.yml

```diff
@@ -61,14 +61,17 @@
               - name: interval
                 value: '28s'
 
-      - id: dummy-no-attrs
+      - id: dmy-no-attrs
```

Comment:
would prefer not to use `dmy` here - maybe `resource-no-attrs` or `test-no-attrs` or similar